src/database/migrations/20240404102919-upload_session_role_permissions.js

```diff
@@ -0,0 +1,136 @@
+'use strict'
+
+require('module-alias/register')
+const userRequests = require('@requests/user')
+require('dotenv').config()
+const common = require('@constants/common')
+const Permissions = require('@database/models/index').Permission
+const RolePermissionMapping = require('@database/models/index').RolePermission
+
+const getPermissionId = async (module, request_type, api_path) => {
+	try {
+		const permission = await Permissions.findOne({
+			where: { module, request_type, api_path },
+		})
+		if (!permission) {
+			throw error
+		}
+		return permission.id
+	} catch (error) {
+		throw error
+	}
+}
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+	async up(queryInterface, Sequelize) {
+		try {
+			const rolePermissionsData = [
+				{
+					role_title: common.MENTEE_ROLE,
+					permission_id: await getPermissionId(
+						'cloud-services',
+						['POST', 'DELETE', 'GET', 'PUT', 'PATCH'],
+						'/mentoring/v1/cloud-services/*'
```

Comment:
@sumanvpacewisdom Why this permission required?

Reply:
only session manager can create the session bulk upload right , so i have given permission only for session manager role